var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -324,12 +324,6 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     # see https://trilinos.org/pipermail/trilinos-users/2015-March/004731.html
     # and https://trilinos.org/pipermail/trilinos-users/2015-March/004802.html
     conflicts("+superlu-dist", when="+complex+amesos2")
-    # https://github.com/trilinos/Trilinos/issues/2994
-    conflicts(
```

Comment:
I looked up this issue. @jrood-nrel was it ever actually resolved?

Reply:
If it was resolved, it needs to be a `when="@:XXX +stk platform=darwin` rather than simply deleted.

Reply:
I will try it today. My guess it was resolved by:
https://github.com/spack/spack/blob/1340995249cd535c8353ab8f2a3c8352777e24c1/var/spack/repos/builtin/packages/trilinos/package.py#L497-L498
Currently this conflict is in direct conflict of those lines though, so this conflict should have probably been removed when these lines were added.